plugins/kubecarrier.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kubecarrier
```

Comment:
I'd recommend to call this `carrier` instead to avoid the stuttering "**kube**ctl **kube**carrier"